### PR TITLE
Added new `showImage` property to the error component

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-error-show-image-property](https://developer.blackbaud.com/skyux?_sp.spa=pr-error-show-image-property)

File: src/app/components/error/index.html [hunk 1]
```diff
@@ -235,6 +235,14 @@ <h3>Guidelines</h3>
       Specifies an error type to use a set of pre-defined values for the image, title, and description. The valid options are <sky-code>broken</sky-code>,
       <sky-code>construction</sky-code>, <sky-code>notfound</sky-code>, and <sky-code>security</sky-code>. This property accepts <sky-code>string</sky-code> values.
     </sky-demo-page-property>
+
+    <sky-demo-page-property
+      propertyName="showImage"
+      defaultValue="true"
+      [isOptional]="true"
+    >
+      Indicateds whether to show the error image. This property accepts <sky-code>boolean</sky-code> values.
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/components/error/index.html]: Indicates whether to display

Comment (Blackbaud-TrevorBurch) [reply src/app/components/error/index.html]: Done

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-error-show-image-property](https://developer.blackbaud.com/skyux?_sp.spa=pr-error-show-image-property)

**Pull request merged**